src/GamePersonnel.jsx

```diff
@@ -8,51 +8,56 @@ import MoneyIcon from '@mui/icons-material/MonetizationOn';
 import AddPersonIcon from '@mui/icons-material/PersonAdd';
 import RemoveIcon from '@mui/icons-material/RemoveCircle';
 import Collapse from '@mui/material/Collapse';
-import { transactPersonnel } from './dry/firebase';
+import { transactPersonnel } from './API';
 
 function GamePersonnel(props) {
+	const { userId, player, gameId } = props
   const [personnelType, setPersonnelType] = useState(''); 
   const [personnelCount, setPersonnelCount] = useState(0);
 
   let personnelCost = 0;
   if (personnelType && personnelCount >= 0) {
     personnelCost = personnelCount * personnelCosts[personnelType];
   } else  if (personnelCount < 0){
-    personnelCost = `Terminate ${-(personnelCount)} ${personnelType}`;
```

Comment:
oh shoot, we are using that to display.  I thought it was only for logic, not for logic and copy.  Thats why all that NaN logic was around.  I can revert this and make it work with the shifting type, but I was hoping it would always be a number

Reply:
We can use a less confusing way to display the terminate personnel condition. But I do want to confirm visually to the user that is what's happening.